### DISCUSSION TITLE
Selective channel loading should find signals in nested channels

### DISCUSSION THREAD

**Discussion by VorobiovM** [danielhrisca/asammdf#566] [Ideas]
Hello, I've been working with this library for a while and it's been a great pleasure.
Recently, I tried filtering channels to gain faster loading speeds, unfortunately the channel I am looking for is inside a nested structure of channels.
I noticed that in the library filters only the channels that are referenced by the channel group and later by other linked channels on *cn_cn_next* address.
What if there would be an option to select a channel referenced inside the *cn_composition* address?

Any thoughts are appreciated 😄

**Comment by danielhrisca** [comment]
@VorobiovM good catch. The fix should be available in the development branch (6.4.5.dev2)
